File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/MailTemplateConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class MailTemplateConfigModel extends ConfigModel {
	public String title;//标题
	public String sender;//发件人
	public String content;//正文
	public ArrArrInt attachment;//附件（动态邮件不需要附件）
}

File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/MailTemplateConfigModel.cs
namespace ConfigSystem
{
﻿    public class MailTemplateConfigModel :ConfigModel {
        public string title;//标题
        public string sender;//发件人
        public string content;//正文
        public System.Collections.Generic.List<System.Collections.Generic.List<int>> attachment;//附件（动态邮件不需要附件）
    }
}